packages/react-positioning/src/usePopper.ts

```diff
@@ -70,222 +39,63 @@ function usePopperOptions(options: PopperOptions, popperOriginalPositionRef: Rea
     overflowBoundary,
     pinned,
     position,
+    unstable_disableTether: disableTether,
     positionFixed,
-    // eslint-disable-next-line @typescript-eslint/naming-convention
-    unstable_disableTether,
   } = options;
 
-  const isRtl = useFluent().dir === 'rtl';
-  const placement = getPlacement(align, position, isRtl);
-  const strategy = positionFixed ? 'fixed' : 'absolute';
-
-  const offsetModifier = React.useMemo(
-    () =>
-      offset
-        ? {
-            name: 'offset',
-            options: { offset: isRtl ? applyRtlToOffset(offset) : offset },
-          }
-        : null,
-    [offset, isRtl],
-  );
+  const { dir } = useFluent();
+  const isRtl = dir === 'rtl';
+  const strategy: Strategy = positionFixed ? 'fixed' : 'absolute';
 
   return React.useCallback(
     (
-      target: HTMLElement | PopperJs.VirtualElement | null,
+      target: HTMLElement | PositioningVirtualElement | null,
       container: HTMLElement | null,
       arrow: HTMLElement | null,
-    ): PopperJs.Options => {
+    ) => {
       const scrollParentElement: HTMLElement = getScrollParent(container);
       const hasScrollableElement = scrollParentElement
         ? scrollParentElement !== scrollParentElement.ownerDocument?.body
         : false;
```

Comment:
As we don't use scrollParentElement can we extract this to hasScrollableElement function?